Source/Classes/AnimatedImages/PINAnimatedImage.m

```diff
@@ -91,8 +91,8 @@ - (NSTimeInterval)minimumFrameInterval
     // Scales the frame delays by `kGreatestCommonDivisorPrecision`
     // then converts it to an UInteger for in order to calculate the GCD.
     NSUInteger scaledGCD = lrint([self durationAtIndex:0] * kGreatestCommonDivisorPrecision);
-    for (NSUInteger durationIdx = 0; durationIdx < self.frameCount; durationIdx++) {
-        Float32 duration = [self durationAtIndex:durationIdx];
+    for (NSUInteger durationIdx = 1; durationIdx < self.frameCount; durationIdx++) {
+        CFTimeInterval duration = [self durationAtIndex:durationIdx];
```

Comment:
Is this change a separate bug fix?

Reply:
Not a bug actually, it's just because we don't need to calculate gcd of index 0 again, because we already calculate at L93 when initialize `scaledGCD`.